lib/linux.js

```diff
@@ -32,6 +44,12 @@ const appsList = [
 		set: ['--bg-scale', '%s']
 	},
 	{
+		cmd: 'nitrogen',
+		set: ['--set-zoom-fill', '--save', '%s'],
+		get: ['-n', 's/^file=//p', `${process.env.HOME}/.config/nitrogen/bg-saved.cfg`],
```

Comment:
Use `require('os').homedir()` instead of the environment variable. Should also use `path.join`.